components/function-controller/internal/controllers/serverless/build_resources.go

```diff
@@ -131,3 +133,13 @@ func (r *FunctionReconciler) mergeLabels(labelsCollection ...map[string]string)
 	}
 	return result
 }
+
+func (r *FunctionReconciler) getPackageConfigVolumeMountsForRuntime(rtm serverlessv1alpha1.Runtime) []corev1.VolumeMount {
```

Comment:
Maybe I'm wrong but I can't find a place where this function is used. Is this an orphan function?

Reply:
yes you're right. It was refactored in the meantime and moved. I will clean up